### DISCUSSION TITLE
Is there any way to know the meaning of the queue:list functions?

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3871] [Q&A]
good, I'm working with dkan2, I execute a series of steps, when creating harvest or eliminating them, and they have to do with the processes of the queue: list, but I don't see in the dkan documentation, what is the meaning and the process of each one, some kind of definition, I would be grateful if you help me defining what each one means and what is the process that it does, thank you very much

![image](https://user-images.githubusercontent.com/22879539/200932021-0e687bd1-5247-4968-9e3a-9c531efd1391.png)

**Comment by janette** [comment]
| Queue | Description |
| -- | -- |
| datastore_import | This queue handles fetching remote files and parsing the data into datastore tables | 
| post_import | This queue will apply data-dictionary definitions and indexes to the datastore tables |
| resource_purger | This queue will purge unneeded resource revisions |
| orphan_reference_processor | This queue deals with the [referenced nodes](https://demo.getdkan.org/modules/contrib/dkan/docs/guide_metastore_properties.html) that have been orphaned by a change to the dataset |
| orphan_resource_remover | Deletes orphaned resources belonging to deleted distributions |

**Reply by tridoxx** [reply]
thank so much
